docs/contents.md

```diff
@@ -13,6 +13,7 @@ The **SLA Management** solution pack contains the following resources.
 
 ## Connector
 
+
 | Name           | Description                                                                                                                                           |
 |:---------------|:------------------------------------------------------------------------------------------------------------------------------------------------------|
 | SLA Calculator | Calculates SLA due date based on locale and work hours. This connector needs soar framework solution pack for supporting playbooks and module changes |
```

Comment:
The description of the connector need to be changed. It no longer has dependency on the SOAR framework. Connector now needs SLA Management SP for supporting playbooks and module